__tests__/api.test.js

```diff
@@ -94,4 +93,107 @@ describe('4. GET /api/articles/:article_id', () => {
+            "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700", 
+            "author": "butter_bridge", 
+            "created_at": "2020-07-09T20:11:00.000Z", 
+            "title": "Living in the shadow of a great man", 
+            "topic": "mitch", 
+            "votes": 100,
+            "comment_count": 11}
+            const articleDB = articles.find((articleOne) => articleOne.article_id === 1);
+            expect(articleDB).toEqual(articleOne)
+    })
+})
+    test("should return articles in descending order of date created", () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            const {articles} = body
+        const orderedArticles = [...articles]
+        orderedArticles.sort((a, b) => a.created_at - b.created_at);
+        expect(orderedArticles).toEqual(articles)
+})
+})
+})
+
+describe('6. GET /api/articles/:article_id/comments', () => {
+    test('should return the required outputs for a single hard coded example AND return 11 expected comments', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            const articleOne = {"article_id": 1, "author": "butter_bridge", "body": "The beautiful thing about treasure is that it exists. Got to find out what kind of sheets these are; not cotton, not rayon, silky.", "comment_id": 2, "created_at": "2020-10-31T03:03:00.000Z", "votes": 14}
+            const {articles} = body
+            expect(articles.length).toBe(11)
+            expect(articles[0]).toEqual(articleOne)  
+    });
+});
+    test("should return most recent comments first", () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            const {articles} = body
+        const orderedArticles = [...articles]
+        orderedArticles.sort((a, b) => a.created_at - b.created_at);
+        expect(orderedArticles).toEqual(articles)
+})
+})
+
+})
+
+describe('7. POST /api/articles/:article_id/comments', () => {
+    test('should return the comment and ensure post is made with 2 new objects', () => {
+        return request(app)
+            .post('/api/articles/1/comments')
+            .send({
+                username: "butter_bridge", 
+                body: "sent a new comment"
+            })
+                .expect(201)
+            .then(({ body }) => {
+                expect(body.comment).toBe("sent a new comment");
+            
+            return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({body}) => {
+                const {articles} = body
+                expect(articles[11].body).toEqual('sent a new comment')
+                expect(articles[11].author).toEqual('butter_bridge')
+
+            })
+    })

```

Comment:
I can only see one test here which is not sufficient for this endpoint. 

Here are some ideas of things you could test for: 

posting to an article which doesn't exist 404
posting using an id which isn't in the correct format 400
posting using a username which does not exist in the db 404
posting using the wrong fields/empty object Ie { notUsername: "butter_bridge", notBody: "validComment" } 400

you may also want to test that any extra properties on the requestObject are ignored - 201. Ie {
                username: "butter_bridge", 
                body: "sent a new comment",
                maliciousSQLAttempt: "DROP DATABASE IF EXISTS nc_news" 
            }